pyproject.toml

```diff
@@ -1,173 +1,173 @@
-
-[tool.coverage.report]
-exclude_lines = [
-    "pragma: no cover",
-    "def __repr__",
-    "if selfdebug",
-    "raise AssertionError",
-    "raise NotImplementedError",
-    "if 0:",
-    "if __name__ == .__main__.:",
-    "@(abc.)?abstractmethod",
-    "if TYPE_CHECKING:",
-    "def _repr_pretty_",
-    "self._pim_instance",
-    "self._pim_file_service",
-    "def _launch_ensight_with_pim"]
-
-ignore_errors = true
-show_missing = true
-
-[tool.coverage.html]
-show_contexts = true
-
-[tool.black]
-line-length = 100
-src_paths = ["src/ansys", "doc"]
-
-[tool.pytest.ini_options]
-minversion = "7.1"
-testpaths = [
-    "tests",
-]
-addopts = "--setup-show --cov=ansys.pyensight.core --cov-report html:coverage-html --cov-report term --cov-config=.coveragerc --capture=tee-sys --tb=native -p no:warnings"
-markers =[
-    "integration:Run integration tests",
-    "smoke:Run the smoke tests",
-    "unit:Run the unit tests",
-    ]
-norecursedirs = ".git .idea"
-filterwarnings = "ignore:.+:DeprecationWarning"
-
-
-[tool.codespell]
-ignore-words-list = "ro, te, pres"
-quiet-level = 3
-skip = "*.pyc,*.xml,*.gif,*.png,*.jpg,*.js,*.html,*.svg,tests/ensigth_api_test_assets.txt"
-
-[tool.isort]
-profile = "black"
-skip_gitignore = true
-force_sort_within_sections = true
-line_length = 100
-default_section = "THIRDPARTY"
-filter_files = true
-src_paths = ["ansys", "doc", "tests"]
-
-[tool.bandit]
-level = "HIGH"
-number = "3"
-recursive = true
-exclude = ["venv/*", "tests/*"]
-
-[tool.mypy]
-python_version = 3.9
-strict = false
-namespace_packages = true
-explicit_package_bases = true
-exclude = ["tests/", "^doc"]
-no_site_packages = true
-follow_imports = "skip"
-
-[[tool.mypy.overrides]]
-module = ["grpc",
- "grpc_health",
- "grpc_bindings",
- "requests",
- "ansys",
- "ansys.*",
- "docker",
- "dill",
- "IPython.display",
- "enve",
- "urllib3",
- "ensight",
- "numpy",
- "PIL",
- "simple_upload_server.*"
-]
-ignore_missing_imports = true
-
-[pydocstyle]
-convention = "numpy"
+[build-system]
+requires = [
+    "flit_core>=3.2,<4"
+]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "ansys-pyensight-core"
+version = "0.8.0.dev0"
+description = "A python wrapper for Ansys EnSight"
+readme = "README.rst"
+requires-python = ">=3.9,<4"
+license = {file = "LICENSE"}
+authors = [{name = "ANSYS, Inc.", email = "pyansys.core@ansys.com"}]
+maintainers = [{name = "ANSYS, Inc.", email = "pyansys.core@ansys.com"}]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+
+dependencies = [
+    "importlib-metadata>=4.0; python_version<='3.8'",
+    "ansys-api-pyensight==0.5.0",

```

Comment:
why this jump?
we can make the api release as a 0.3.x or a 0.4.0 at most, but I'd do 0.4.0 when we release 242. So right now to test it it could be a 0.3.x.
The api repo is kind of disconnected from this repo so we can follow, more or less, the order we like

Reply:
Actually, I think I would have preferred 0.8.0, but I happened to commit the version as 0.5.0 (in the api git repo).